core/domain/question_services.py

```diff
@@ -213,7 +222,177 @@ def update_question(
             question.
         commit_message: str or None. A description of changes made to the
             question.
+
+    Raises:
+        ValueError: No commit message was provided.
     """
+    if not commit_message:
+        raise ValueError(
+            'Expected a commit message, received none.')
+
     updated_question = apply_change_list(question_id, change_list)
     _save_question(
         committer_id, updated_question, change_list, commit_message)
+    create_question_summary(question_id, committer_id)
+
+
+def create_question_summary(question_id, creator_id):
+    """Creates and stores a summary of the given question.
+
+    Args:
+        question_id: str. ID of the question.
+        creator_id: str. The user ID of the creator of the question.
+    """
+    question = get_question_by_id(question_id, strict=False)
```

Comment:
Same as above. If you do strict=False you *must* handle the None case. If you don't expect a "not found" error to happen, don't include that arg.

Reply:
Done.